src/types/block.ts

```diff
@@ -31,7 +31,7 @@ export type Block<
   /** Block hash or `null` if pending */
   hash: blockTag extends 'pending' ? null : Hash
   /** Logs bloom filter or `null` if pending */
-  logsBloom: blockTag extends 'pending' ? null : Hex
+  logsBloom: blockTag extends 'pending' ? null : Hex | ''
```

Comment:
I would assume that RPC Providers / Nodes will coerce zero data as undefined as they do with other fields. 

```suggestion
  logsBloom?: Hex | null | undefined
```